stopify/src/runtime/suspend.ts

```diff
@@ -57,7 +57,6 @@ export class RuntimeWithSuspend {
    * @param force forces a suspension when `true`.
    */
   suspend(force?: boolean): void {
-    if (this.rts.isSuspended) { debugger; }
```

Comment:
Don't refactor...

Reply:
This should’ve been in the refactoring PR. I’ll move it to a new commit. 
Random debugger statements affect performance.